cob_obstacle_distance/scripts/test_obstacle_publisher_node.py

```diff
@@ -79,19 +79,22 @@
     y = CollisionObject()
     y.id = "Funny Mesh"
     y.header.frame_id = root_frame
-    y.type.db = "package://cob_gazebo_objects/Media/models/milk.dae"
-    y.operation = CollisionObject.ADD
+     # y.type.db = "package://cob_gazebo_objects/Media/models/milk.dae"
+    y.type.db = "package://cob_twist_controller/files/torus_0_25_inner_rad.stl"
+    # y.type.db = "package://cob_twist_controller/files/untitled.dae"
+    # y.operation = CollisionObject.ADD
+    y.operation = CollisionObject.REMOVE
```

Comment:
Please undo the changes in this file as it is meant to be a basic example.
Add a new one for adding the torus! If you do so, add the mesh file, too!